drivers/ethos/ethos.c

```diff
@@ -198,18 +198,18 @@ static size_t iolist_count_total(const iolist_t *iolist)
     return result;
 }
 
-static void _write_escaped(uart_t uart, uint8_t c)
+static void _write_escaped(uart_t uart, const uint8_t c)
```

Comment:
Adding `const` here doesn't really make sense and doesn't add anything, as `c` is not a pointer.

Reply:
I guess I should practice what I preach. I though it would complain below when I assigned it's address to a `const uint8_t *`.